src/traffic_ctl_jsonrpc/CtrlPrinters.cc

```diff
@@ -0,0 +1,318 @@
+                 [&](RecordLookUpResponse::RecordParamInfo::ConfigMeta const &meta) {
+                   std::cout << ts::bwprint(text, "{:16s}: {}\n", "Access Control ", rec_accessof(meta.accessType));
+                   std::cout << ts::bwprint(text, "{:16s}: {}\n", "Update Type ", rec_updateof(meta.updateType));
+                   std::cout << ts::bwprint(text, "{:16s}: {}\n", "Update Status ", meta.updateStatus);
+                   std::cout << ts::bwprint(text, "{:16s}: {}\n", "Source ", rec_sourceof(meta.source));
+
+                   std::cout << ts::bwprint(text, "{:16s}: {}\n", "Syntax Check ", meta.checkExpr);
+                 },
+                 [&](RecordLookUpResponse::RecordParamInfo::StatMeta const &meta) {
+                   // This may not be what we want, as for a metric we may not need to print all the same info. In that case
+                   // just create a new printer for this.
+                   std::cout << ts::bwprint(text, "{:16s}: {}\n", "Persist Type ", meta.persistType);
+                 },
+               },
+               recordInfo.meta);
+
+    std::cout << ts::bwprint(text, "{:16s}: {}\n", "Overridable", (recordInfo.overridable ? "yes" : "no"));
+    std::cout << ts::bwprint(text, "{:16s}: {}\n", "Version ", recordInfo.version);
+    std::cout << ts::bwprint(text, "{:16s}: {}\n", "Order ", recordInfo.order);
+    std::cout << ts::bwprint(text, "{:16s}: {}\n", "Raw Stat Block ", recordInfo.rsb);
+  }
+}
+
+void
+RecordDescribePrinter::write_output_pretty(RecordLookUpResponse const &response)
+{
+  std::string text;
+
+  write_output_legacy(response);
+  print_record_error_list(response.errorList);
+}
+//------------------------------------------------------------------------------------------------------------------------------------
+void
+GetHostStatusPrinter::write_output(YAML::Node const &result)
+{
+  auto response = result.as<RecordLookUpResponse>();
+  std::string text;
+  for (auto &&recordInfo : response.recordList) {
+    std::cout << ts::bwprint(text, "{} {}\n", recordInfo.name, recordInfo.currentValue);
+  }
+  for (auto &&e : response.errorList) {
+    std::cout << ts::bwprint(text, "Failed  to fetch {}\n", e.recordName);
+  }
+}
+
+//------------------------------------------------------------------------------------------------------------------------------------
+void
+SetHostStatusPrinter::write_output(YAML::Node const &result)
+{
+  // do nothing.
+}
+//------------------------------------------------------------------------------------------------------------------------------------
+
+void
+CacheDiskStoragePrinter::write_output(YAML::Node const &result)
+{
+  // do nothing.
+  if (!is_format_legacy()) {
+    write_output_pretty(result);
+  }
+}
+void
+CacheDiskStoragePrinter::write_output_pretty(YAML::Node const &result)
+{
+  std::string text;
+  auto my_print = [&text](auto const &disk) {
+    std::cout << ts::bwprint(text, "Device: {}\n", disk.path);
```

Comment:
Why not just:
```
std::cout << "Device: " << disk.path << '\n';
```

Reply:
Yes, for the sake of consistency I did that everywhere(in traffic_ctl), probably for legacy and simple output like this one I can just switch to the classic.